src/me/StevenLawson/TotalFreedomMod/TFM_Util.java

```diff
@@ -993,7 +993,7 @@ public static void reportAction(Player reporter, Player reported, String report)
         {
             if (TFM_AdminList.isSuperAdmin(player))
             {
-                playerMsg(player, ChatColor.RED + "[REPORTS] " + ChatColor.GOLD + reporter.getName() + " has reported " + reported.getName() + " for " + report);
+                playerMsg(player, ChatColor.DARK_RED + "[REPORTS] " + ChatColor.RED + reporter.getName() + " has reported " + reported.getName() + " for " + report);
```

Comment:
If I remember correctly, Dark red can be rather difficult to read on a lot of screens, and is not often the ideal one to use.